packages/ui/package.json

```diff
@@ -25,7 +25,11 @@
     "typecheck": "tsc --noEmit --project ./tsconfig.json"
   },
   "exports": {
-    ".": "./dist-web/web.js",
+    "./web": {
```

Comment:
Let's try and keep the naked path, and only have `/native` for native, as react is `react` and not `react/web`

Reply:
I changed it, thanks 👍